security.qmd

```diff
@@ -1,3 +1,34 @@
 # Security {{< iconify fa6-solid lock >}} {#sec-security}
 
-xxx
\ No newline at end of file
+## Package security
+
+Keeping secrets a secret is an important task and should not be an afterthought - but instead considered from the beginning of a package.
+
+In general, the rOpenSci Development Guide has good advice on [package security](https://devguide.ropensci.org/package-development-security-best-practices.html).
+
+As time goes on we'll document here if we have any strong opinions about security related aspects of packages.
+
+## Data
+
+Software created in the WILDS may touch PII/PHI data.
```

Comment:
I think that instead of writing sensitive data handling practices here we should link to the sciwiki or something.

Reply:
Okay. Are you thinking we just remove this chapter? Or keep the chapter but simply link out to sciwiki/other resources with very little text in the chapter itself?

Reply:
I think we should keep it and link out. The we could have sections like "how to securely integrate [a WILDS thing] with [a service discussed on sciwiki]"

Reply:
Sounds good!